src/misc/utils.cc

```diff
@@ -18,6 +20,16 @@ extern "C" {
 #include <string>
 #include <vector>
 
+#ifdef __linux__
+#include <sys/prctl.h>
+#endif
+
+#ifdef OPEN_MAX
```

Comment:
We should add a comment here to point out why this is necessary, and where it's used.